public/pages/game.html

```diff
@@ -106,4 +105,5 @@
       </footer>
     </div>
   </body>
+  <script src="../js/scoreboard-rounds.js"></script>
```

Comment:
We should include this block of code inside the body, not outside, and add type module as a property of script here.

```suggestion
  <script src="../js/scoreboard-rounds.js" type="module"></script>
```

Reply:
The script was added outside the body because it will need the body loaded previously (button that starts the game) to set the counter number of rounds